run_endpoint.sh

```diff
@@ -13,7 +13,7 @@ LOGLEVEL=2
 /setup.sh
 
 # Unless noted otherwise, test cases use HTTP/0.9 for file transfers.
-PROTOCOL="hq-${DRAFT}"
+PROTOCOL="hq-interop"
```

Comment:
I am not sure this will work. HQSession expects hq-29 I believe. I'll merge and check the results